src/de.rs

```diff
@@ -132,6 +133,17 @@ where
         }
     }
 
+    #[inline]
+    fn parse_raw_value(&mut self) -> Result<Reference<'de, '_, [u8]>> {
+        match &self.state {
+            State::Value => {
```

Comment:
According to the documentation, `LIST_ITEM` can also be a raw blob sequence. Could you also support that as well?

https://github.com/strongswan/strongswan/blob/6.0.1/src/libcharon/plugins/vici/README.md#message-format
> Types having a value (KEY_VALUE and LIST_ITEM) have a raw blob sequence, prefixed with a 16-bit network order length. The blob follows the type or the name tag if available, the length defined by the length field does not include the length field itself.
>
> The interpretation of any value is not defined by the message format; it can take arbitrary blobs. The application may specify types for specific keys, such as strings or integer representations. The vici plugin currently uses non-null terminated strings as values only; numbers get encoded as strings.

```suggestion
            State::Value | State::ListItem(_) => {
```

Reply:
Both of KEY_VALUE and LIST_ITEM are always blobs. Till now, I didn't face any situation where I needed Vec<Vec<u8>>. I think that can be solved in a way mentioned in this issue:

serde-rs/bytes#29

I will test this when I get time.